### ISSUE TITLE
Using `tiles.setTileAt` does not update the tile in the 3D rendered space

### ISSUE THREAD

**Issue by megabyte6** [AqeeAqee/pxt-raycasting#14]
Recently I was helping someone with a game and they were looking to change the texture on a tile when interacting with another element in-game. We got as far as changing the texture using the [set tile at location] block to change the texture of the tile but it only changes in the tilemap mode and not in the 3D render mode. Would it be possible to detect the change in the tilemap and update the 3D rendered tilemap accordingly?

**Comment by megabyte6** [comment]
@AqeeAqee I implemented the fix that I thought was best since you can't inject code into a built-in function. I added the function in the 3D Render block menu that matches the one built in the Scene menu. I tried to follow your code style and used similar formatting for the doc page. I did my best to match both your doc pages as well as the original [set tile at location] block on MakeCode's docs page. If there's anything you want changed just let me know and I'd be happy to make the change.

Additionally, I was wondering if it was alright to update the `targetVersions` in `pxt.json`. MakeCode Arcade automatically updated the `target` so I figured it would be good to update the rest as well. I'm not sure if you kept the version at that particular release because of some compatibility reason so I only pushed the change to `targetVersions.target` that was automatically done and nothing else. If there is no reason for keeping it at the current version, I'd be happy to update that as well.

**Comment by AqeeAqee** [comment]
Fixed in #17 by @megabyte6

**Comment by megabyte6** [comment]
@AqeeAqee sorry for asking if this is mentioned somewhere obvious but how are the updates pushed to MakeCode Arcade? From what I found [here](https://makecode.com/extensions/versioning), it seems that you'd need to add a tag to the current commit (probably something like v0.3.19). I also noticed you add a release with a `.uf2` file each time. Is that also necessary or simply something else you decided to do?
